playwright.config.js

```diff
@@ -24,7 +24,12 @@ module.exports = defineConfig({
   /* Opt out of parallel tests on CI. */
   workers: process.env.CI ? 9 : 9,
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: 'html',
+  // reporter: 'html',
+  reporter: [
+    ['html', { outputFolder: './playwright-report', open: 'never' }],
+    ['junit', { outputFile: './playwright-report/results.xml' }],
+    ['json', { outputFile: './playwright-report/results.json' }]
```

Comment:
Do we need the `html` and `junit` options? I can see on `json` (`playwright-report/results.json`) being used above?

Reply:
@pokhiii  the junit is not required hence i have removed it ['junit', { outputFile: './playwright-report/results.xml' }],
  
  ['html', { outputFolder: './playwright-report', open: 'never' }], is required for generation of videos's and traces